packages/repack/src/commands/common/setupInteractions.ts

```diff
@@ -23,16 +57,83 @@ export function setupInteractions(
         case 'c':
           process.exit();
           break;
+
         case 'z':
           process.emit('SIGTSTP', 'SIGTSTP');
           break;
       }
-    } else if (name === 'r') {
-      handlers.onReload?.();
-      logger.info('Reloading app');
-    } else if (name === 'd') {
-      handlers.onOpenDevMenu?.();
-      logger.info('Opening developer menu');
+    } else {
+      const interaction = plainInteractions[name];
+
+      if (interaction) {
+        const {
+          action,
+          postPerformMessage,
+          helpName,
+          actionUnsupportedExplanation,
+        } = interaction;
+
+        if (action && actionUnsupportedExplanation === undefined) {
+          logger.info(postPerformMessage);
+
+          action();
+        } else {
+          logger.warn(
+            `${helpName} is not supported ${actionUnsupportedExplanation ?? 'by the used bundler'}`
+          );
+        }
+      }
     }
   });
+
+  // since now Re.pack officially supports RN >= 0.73, it is sure that RN
+  // has the capability of the new debugger
+  const hasExperimentalDebuggerSupport = process.argv.includes(
+    EXPERIMENTAL_DEBUGGER_FLAG
+  );
+
+  const plainInteractions: Record<string, Interaction | undefined> = {
+    r: {
+      action: handlers.onReload,
+      postPerformMessage: 'Reloading app',
+      helpName: 'Reload app',
+    },
+    d: {
+      action: handlers.onOpenDevMenu,
+      postPerformMessage: 'Opening developer menu',
+      helpName: 'Open developer menu',
+    },
+    j: {
+      action: handlers.onOpenDevTools
+        ? () => {
+            if (hasExperimentalDebuggerSupport) {
+              handlers.onOpenDevTools!();
+            } else {
+              logger.warn(
+                `DevTools require the '${EXPERIMENTAL_DEBUGGER_FLAG}' flag to be passed to the bundler process`
+              );
+            }
+          }
+        : undefined,
+      postPerformMessage: 'Opening DevTools',
+      helpName: 'Open DevTools',
+      actionUnsupportedExplanation: hasExperimentalDebuggerSupport
+        ? undefined
+        : `${EXPERIMENTAL_DEBUGGER_FLAG} was not passed`,
+    },
+  };
+
+  console.log(blueBright('You can use the following keystrokes:'));
+  for (const [key, interaction] of Object.entries(plainInteractions)) {
+    const isSupported =
+        interaction?.actionUnsupportedExplanation === undefined &&
+        interaction?.action !== undefined,
+      text = `${key}: ${interaction?.helpName}${isSupported ? '' : yellow(` (unsupported${interaction?.actionUnsupportedExplanation ? `, ${interaction.actionUnsupportedExplanation}` : 'by the current bundler'})`)}`;
+
+    console.log(isSupported ? text : italic(text));
+  }
+
+  console.log('');
+  console.log('Press ctrl+c or ctrl+z to quit the dev server');
+  console.log('');
```

Comment:
use logger instead of console

Reply:
As we spoke, the `ctx.logger` does not write anything at this stage.

Reply:
use `logger` but dont pass `ctx.logger` in `start.ts` 👍